ibft/controller/controller_sync.go

```diff
@@ -33,9 +34,18 @@ func (i *Controller) processSyncQueueMessages() {
 
 // ProcessSyncMessage processes sync messages
 func (i *Controller) ProcessSyncMessage(msg *network.SyncChanObj) {
+	i.logger.Info("ProcessSyncMessage: processing message")
```

Comment:
i think we can remove this log, no?

Reply:
Sure, I needed it for debugging and will delete it before merging